effekt/jvm/src/main/scala/effekt/EffektConfig.scala

```diff
@@ -22,6 +22,14 @@ class EffektConfig(args: Seq[String]) extends REPLConfig(args) {
     default = Some(false)
   )
 
+  val trace: ScallopOption[String] = choice(
+    choices = Seq("text", "json"),
+    name = "trace",
+    descr = "Trace the time spent in each compilation phase.",
+    default = Some("text"),
```

Comment:
Does this enable tracing by default?

Reply:
No, I don't think so, see [here](https://github.com/effekt-lang/effekt/blob/98a0e86b928a59f5cb0515037e47019f41082074/effekt/shared/src/main/scala/effekt/context/Context.scala#L74). But looking at it again, it is a bit weird -- supplying a default value for an optional argument.

Reply:
Does this have the right behavior?

Like `--trace` without `text` or `json` will be tracing in `text` mode?

Reply:
No, it doesn't. From the `choice` docstring:

> Default value to use if option is not found in input arguments (if you provide this, you can omit the type on method).

Maybe `choice` is not the right choice for this behaviour. I agree that only having to supply `-t` instead of `-t text` is more desirable.

Reply:
In the end this is not too important, but then I would default it to `None`.